frontend/src/views/SettingWorkspaceVCSDetail.vue

```diff
@@ -195,15 +200,9 @@ export default {
       const payload = (event as CustomEvent).detail as OAuthWindowEventPayload;
       if (isEmpty(payload.error)) {
         if (vcs.value.type == "GITLAB_SELF_HOST") {
-          const oAuthConfig: OAuthConfig = {
-            endpoint: `${vcs.value.instanceUrl}/oauth/token`,
-            applicationId: state.applicationId,
-            secret: state.secret,
-            redirectUrl: redirectUrl(),
-          };
           store
-            .dispatch("gitlab/exchangeToken", {
-              oAuthConfig,
+            .dispatch("oauth/exchangeToken", {
```

Comment:
oauth/exchangeVCSToken

otherwise, we need refactor when we add oauth support with a different system